implants/Cargo.toml

```diff
@@ -47,6 +47,8 @@ network-interface = "1.0.1"
 nix = "0.26.1"
 notify = "6.1.1"
 object = "0.31.1"
+openssl = "0.10.55"
```

Comment:
Sir `openssl` is banned.

Reply:
Out of curiosity, why?

Reply:
We just got rid of OpenSSL and shaved 7MB, we're using rusttls for TLS now.